lib/tags/note.js

```diff
@@ -41,6 +41,8 @@ module.exports = function note (hexo, args, content) {
 
   return Promise.resolve(
     toMarkdown(content)
+    .replace('–', '--')
+    .replace('—', '---')
```

Comment:
![](https://media.giphy.com/media/pVAMI8QYM42n6/giphy-tumblr.gif)

Reply:
The values to be replaced _look_ exactly the same but they are distinct: an em-dash (mutton) and an n-dash (nut).

Reply:
Oh boy 😂